examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
<p>Nice touch adding the mean ESS per second! Do you think it'd be useful to add that to ArviZ natively?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<blockquote>Do you think it'd be useful to add that to ArviZ natively?</blockquote><p>...Meh. It's unfortunately very dependent on other stuff, like burn-in, what architecture you're on, chains vs. cores, etc. In the end, just comparing sampling time is probably the better option.</p><p> </p><p>Might be worth thinking about some more comparable measure. (Floating-point operations per ES? Is that possible?)</p>

Reply:
Ah yeah, good point 🤔 However, if you think of helpful measures that would help diagnose models, feel free to open issues (or even PRs) on the [ArviZ](https://github.com/arviz-devs/arviz) side!

Reply:
Some smooth recruiting going on here ^^. I approve

Reply:
Don't blow my cover @ricardoV94! 🤫

Reply:
<p>:P</p>